components/layouts/TheMenu/MobileMenu.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <section class="mobile-menu">
+    <nav
+      class="mobile-menu__navigation-links"
+    >
+      <AppLink
+        v-for="link in mainLevelLinks"
+        :key="link.url"
+        class="mobile-menu__link"
+        :class="{ 'mobile-menu__link_active': isActive(link) }"
+        v-bind="link"
+      >
+        <p class="mobile-menu__link-label">
+          {{ link.label }}
+        </p>
+      </AppLink>
+    </nav>
+    <footer class="mobile-menu__footer">
+      <div class="mobile-menu__footer-inner-container">
+        <p class="mobile-menu__footer-text">
+          ©Qiskit | All Rights Reserverd
+        </p>
+      </div>
+    </footer>
+  </section>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator'
+import AppLink from '~/components/ui/AppLink.vue'
+import FooterSection from '~/components/layouts/PageFooter/FooterSection.vue'
```

Comment:
Stay connected has been removed, so can delete all this related to them: imported links, variables, classes, ...